src/pages/basic-management/employee/add/Index.js

```diff
@@ -0,0 +1,195 @@
+import React, { PureComponent, Fragment } from 'react';
+import { connect } from 'dva';
+import router from 'umi/router';
+import {
+    List,
+    Card,
+    Row,
+    Col,
+    Radio,
+    Input,
+    InputNumber,
+    Progress,
+    Button,
+    Icon,
+    Dropdown,
+    Menu,
+    Avatar,
+    Modal,
+    Form,
+    DatePicker,
+    Select,
+    Cascader,
+    message,
```

Comment:
Codacy found an issue: ['message' is defined but never used.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=6006324)